app/indexes/post_index.rb

```diff
@@ -258,6 +266,7 @@ def as_indexed_json(options = {})
       pending:        is_pending,
       deleted:        is_deleted,
       has_children:   has_children,
+      has_pending_replacements: options.key?(:has_pending_replacements) ? options[:has_pending_replacements] : replacements.pending.any?
```

Comment:
Is there a reason that this has an override that bypasses the normal lookup? Is this an optimization?